docs/api.rst

```diff
@@ -18,6 +18,9 @@ Channels
 Listeners
 ---------
 
```

Comment:
Just a heads up that I'm currently having troubles deploying updates to readthedocs. The webhook was accidentally removed, and there seems to be no way of getting it back as django-pgpubsub does not appear as a project on my personal readthedocs (even after transferring this library to my own github. I'm trying to get the owner of Opus10 to help me out here, but until then we won't get readthedocs updates unfortunately.